packages/@aws-cdk/aws-s3-assets/test/asset.test.ts

```diff
@@ -124,6 +124,31 @@ test('"readers" or "grantRead" can be used to grant read permissions on the asse
   });
 });
 
```

Comment:
add a test to verify `asset.bucket` is properly configured (with the key)

Reply:
Why?

The observable behavior (that a `grantRead()` leads to KMS permissions being added) is tested. That's all we need to care about?

Reply:
Got it. Thanks!